k4EDM4hep2LcioConv/src/k4EDM4hep2LcioConv.cpp

```diff
@@ -14,8 +14,12 @@ lcio::LCCollectionVec* convTracks(
   for (const auto& edm_tr : (*tracks_coll)) {
     if (edm_tr.isAvailable()) {
       auto* lcio_tr = new lcio::TrackImpl();
-
-      lcio_tr->setTypeBit(edm_tr.getType());
+      int type = edm_tr.getType();
+      for (int i = 0; i<sizeof(int)*8;i++){
+	if (type & (1<<i)){
+	   lcio_tr->setTypeBit(i);	  
```

Comment:
```suggestion
	   lcio_tr->setTypeBit(i, true);	  
```

explicitly putting the value instead of using the implicit default makes it clearer what is going on here, I think.